yarn.lock

```diff
@@ -18804,7 +18804,7 @@ lodash.find@4.6.0:
   resolved "https://registry.yarnpkg.com/lodash.find/-/lodash.find-4.6.0.tgz#cb0704d47ab71789ffa0de8b97dd926fb88b13b1"
   integrity sha1-ywcE1Hq3F4n/oN6Ll92Sb7iLE7E=
 
-lodash.flatten@^4.2.0, lodash.flatten@^4.4.0:
+lodash.flatten@^4.2.0:
```

Comment:
Is this still left in somewhere else?

Reply:
I have to assume it's still a transitive dependency.  Checking `yarn why`, it's apparently used by `cheerio`, which seems to be used in various places.

I don't think it's particular essential root out this dependency entirely; it's not causing dependabot alerts or warnings or anything.  I just wanted to get rid of our direct usage now that we can just use `flat` and `flatMap`.